src/main/java/frc/robot/subsystems/climb/ClimbSubsystem.java

```diff
@@ -21,7 +28,7 @@ public class ClimbSubsystem extends SubsystemBase {
   private ClimbInputsAutoLogged inputs = new ClimbInputsAutoLogged();
   private ClimbOutputsAutoLogged outputs = new ClimbOutputsAutoLogged();
 
-  // TODO: replace when ramp becomes a thing
+  private RampSubsystem rampSubsystem;
   private BooleanSupplier rampClear = () -> true;
```

Comment:
Is rampClear being set? Why are we passing the whole rampSubsystem to climb rather than supplying isInPosition to this (or some climb specific variant if we don't care about being precisely in position)?